backend/tests/Tests/TypeChecker.Tests.fs

```diff
@@ -84,6 +85,6 @@ let testArguments : Test =
       (("myAnyFn", RT.TVariable "a", S.eInt 5), RT.DInt 5L) ]
 
 
-
+// TODO: add tests around type args
```

Comment:
TODO: add some typeChecker tests around type args (_could_ wait for another PR, if not trivial)

Reply:
leaving the equivalent call-out TODO in language.tests, and removing this TODO